tools/proto-js-plugin/README.md

```diff
@@ -24,6 +24,18 @@ The plugin may then be applied where necessary:
 apply plugin: "io.spine.tools.proto-js-plugin"
 ```
 
+The `protoJs` extension can be used to configure when during the build lifecycle the plugin action
```

Comment:
Can you please rephrase the first sentence. It's way too difficult to comprehend.